Amelia Guise_Assignment 1

```diff
@@ -0,0 +1,342 @@
+#Amelia was here. 
+\No new line at the end of file
+
+#Amelia Guise_Assignment 1
+#(My answers are off- my players seem to have made exponentially more money that in the official answers- which is good for them and bad for me). 
+
+import random
+
+gameStake = 50  
+cards = range(10)
+
+#Defining the class Players
+
+class Player:
+    PlayerID = 0
+    Pot= 0   
+    LastCard=0
+
+#use two input variables to initialise the ID and starting pot of each player
+
+    def __init__(self, inputID, startingPot):
+        self.playerID = inputID
+        self.pot = startingPot
+        
+   #Create a code for selecting a random card. 
+             
+    def play(self,dealerCard):
+        playerCard = random.choice(cards)
+        
+    #Conditional: tests the player's card value against the dealer card.
+    #defines whether the player has won or lost.
+    #tallies up the players pot. 
+
+        
+        if playerCard < dealerCard:
+            self.pot-=gameStake
+            return 'Lose,' + str(playerCard) + 'vs' + str(dealerCard)
+        else:
+            self.pot-=gameStake
```

Comment:
The problem seems to be here. If the player wins, you should add to his total pot using += instead of -=. Currently the player looses either way, leading to very low pots for all players.